modules/mux/targets/riscv/external/CMakeLists.txt

```
@@ -23,6 +23,8 @@ set(RISCV_TOOLCHAIN_DIR "${CA_LLVM_INSTALL_DIR}" CACHE PATH
 
 # Iterate through all 'hal_*' sub-directories.
 file(GLOB HAL_DIR_NAMES FOLLOW_SYMLINKS LIST_DIRECTORIES true "hal_*")
+file(GLOB HAL_DIR_NAMES_EXAMPLES FOLLOW_SYMLINKS LIST_DIRECTORIES true ${PROJECT_SOURCE_DIR}/examples/hals/hal_*)
```

We talked about possibly doing
```suggestion
file(GLOB HAL_DIR_NAMES_EXAMPLES FOLLOW_SYMLINKS LIST_DIRECTORIES true hal_* ${PROJECT_SOURCE_DIR}/examples/hals/hal_*)
```

---

good idea